pythonlib/test/src/mill/pythonlib/HelloWorldTests.scala

```diff
@@ -16,22 +16,22 @@ object HelloWorldTests extends TestSuite {
 
     object qux extends PythonModule {
       override def moduleDeps: Seq[PythonModule] = Seq(foo)
-      override def script = T.source(millSourcePath / "src" / "qux.py")
+      override def mainScript = T.source(millSourcePath / "src" / "qux.py")
       object test extends PythonTests with TestModule.Unittest
     }
   }
 
   val resourcePath = os.Path(sys.env("MILL_TEST_RESOURCE_DIR")) / "hello-world-python"
-
   def tests: Tests = Tests {
-    test("run") {
-      val baos = new ByteArrayOutputStream()
-      val eval = UnitTester(HelloWorldPython, resourcePath, outStream = new PrintStream(baos))
+    // TODO: Giving Some Error Needs to Fix
+    // test("run") {
+    //   val baos = new ByteArrayOutputStream()
+    //   val eval = UnitTester(HelloWorldPython, resourcePath, outStream = new PrintStream(baos))
 
-      val Right(result) = eval.apply(HelloWorldPython.qux.run(Args()))
+    //   val Right(result) = eval.apply(HelloWorldPython.qux.run(Args()))
 
-      assert(baos.toString() == "Hello,  Qux!\n")
-    }
+    //   assert(baos.toString() == "Hello,  Qux!\n")
+    // }
```

Comment:
What's the error here?

Reply:
Actually it's very surprising it was giving the right output as "hello,  Qux!" But i don't know why assert was not working so then after trying manual debugging and some fixes it was not getting fixed 

Hence, i commented 😅

I will show you the exact error in some minutes...

Reply:
It's probably due to me streaming `pip`'s progress to stdout https://github.com/jodersky/mill/blob/3fb0730c08f76c1991c6424d56614555757aa1a7/pythonlib/src/mill/pythonlib/PythonModule.scala#L47 `stdout = os.Inherit`. I think it's useful for the user to see this output, but the caveat is that you can't match exactly on output anymore. You can update the test to be `assert(baos.toString().contains("Hello,  Qux!\n")), like I've also done in my PR.

Reply:
Fixed !